client/state/signup/progress/schema.ts

```diff
@@ -26,7 +26,7 @@ export interface StepState {
 		url: string;
 	};
 	lastUpdated: number;
-	providedDependencies?: string[];
+	providedDependencies?: any;
```

Comment:
Ah sorry can this now be `Dependencies` as well?

Reply:
Oh yes! Renaming the variables makes me feel 😵‍💫